src/guide/code-owners.md

```diff
@@ -53,58 +56,92 @@ This list is intended to make it easier to identify which email list to include
     * `libadt` – Compiler
     * `logging` – Runtime
     * `memory` – Runtime, GC
-    * `metaprogramming` – Runtime
-    * `oops` – Runtime
+    * `metaprogramming` – HotSpot
+    * `oops` – GC, Runtime
     * `opto` – Compiler
-    * `precompiled` – Runtime
+    * `precompiled` – HotSpot
     * `prims` – Runtime, Serviceability
     * `runtime` – Runtime
     * `services` – Runtime
-    * `shark` – Compiler
-    * `trace` – Runtime
-    * `utilities` – Runtime
+    * `utilities` – GC, Runtime
+  * `share/cds` – Runtime
 * `java.base`
-  * Core Libs should almost always be included but LangTools, HotSpot, Security and/or I18n may also be involved.
+  * Core Libs should almost always be included but Java Language, HotSpot, Security and/or I18n may also be involved.
   * `classes`
     * `crypto` – Security
-    * `internal` – HotSpot, Core Libs
+    * `internal`
+      * `access` – Core Libs, Security
+      * `event` – JFR
+      * `foreign` – Core Libs
+      * `icu` – Core Libs
+      * `io` – NIO
+      * `javac` – Java Language
+      * `jimage` – Client Libs
+      * `jmod` – HotSpot
+      * `jrtfs` – Core Libs
+      * `loader` – Core Libs
+      * `logger` – Core Libs
+      * `math` – Core Libs
+      * `misc` – Core Libs, HotSpot
+      * `module` – Core Libs
+      * `org/objectweb` – Core Libs
+      * `org/xml` – Core Libs
+      * `perf` – Runtime
+      * `platform` – HotSpot
+      * `ref` – Core Libs, GC
+      * `reflect` – Core Libs
+      * `util/random` – Core Libs
+      * `util/regex` – Core Libs
+      * `util/xml` – Core Libs
+      * `vm` – HotSpot
     * `invoke` – Core Libs
     * `io` – NIO
-    * `lang` – Core Libs
-    * `launcher` – LangTools
-    * `math` – Core Libs
+    * `java`
+      * `lang` – Core Libs
+      * `math` – Core Libs
+      * `time` – Core Libs
+    * `launcher` – JDK Tools
+    * `META-INF/services` – Core Libs
     * `net` – Net
     * `nio` – NIO
-    * `reflect` – Core Libs
+    * `reflect` – Java Language
     * `security` – Security
+    * `sun/crypto` – Security
```

Comment:
There is no such directory named `sun/crypto` - you probably mean `com/sun/crypto` and `com/sun/security`.

Reply:
Missing several other directories under `sun`.